incubator/hnc/Makefile

```diff
@@ -69,6 +69,7 @@ KREW_CKSM=$(shell sha256sum bin/kubectl-hns.tar.gz | cut -d " " -f 1)
 
 # The directories to run Go command on (excludes /vendor)
 DIRS=./api/... ./cmd/... ./internal/... ./third_party/...
+GOFMT_DIRS=./api ./cmd ./internal ./third_party
```

Comment:
Why are these two different?

Reply:
`gofmt` does not understand the `./api/...` syntax. Instead, it works with the simple: `./api`.
I saw other make targets using the `DIRS` variable `(make vet)`, so I opted to create a new variable for the `go fmt` related make targets.

Reply:
wouldn't `GOFMT_DIRS=$(DIRS:/...=)` work here? Would protect against changes to DIRS in the future. Although, it isn't foolproof.

Reply:
Thanks @rjbez17 that code worked!